releases/cosmwasm/2025-03-Celo-proposal-v1.0.0.md

```diff
@@ -0,0 +1,44 @@
+# Update Celo confirmation height
+
+|                | **Owner**                              |
+| -------------- | -------------------------------------- |
+| **Created By** | @blockchainguyy <ayush@interoplabs.io> |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
```

Comment:
testnet was already done, mark as resolved. Is the conf height param updated there and on stagenet however? If not, we should apply @RiceAndMeet

Reply:
I don't think it was applied on testnet, don't see the proposal. I can submit it for both environment.